docs/easybuild-v5/run_shell_cmd.md

```diff
@@ -8,20 +8,80 @@ This function replaces both the `run_cmd` and `run_cmd_qa` functions, which will
 
 ## Motivation
 
-...
+Over the years `run_cmd` and `run_cmd_qa` accumulated a lot of arguments which sometimes have misleading names.
+Moreover they have two different kinds of return values (depending on whether called with `simple=True` or `simple=False`).
+So a new command `run_shell_cmd` is introduced to replace both, using more direct and natural arguments, and as a bonus,
+better error reporting. In line with `--trace` being set by default, `run_shell_cmd` is now also more verbose by default.
 
 ## High-level overview
 
-...
+First of all `run_shell_cmd` has been designed so the defaults can be used for most situations.
+In that case the main thing to watch out for is the return code, which changed from a tuple `(output, exit_code)`
+to a named tuple with multiple fields, the most prominent ones being:
+
+- `output`: command output, `stdout`+`stderr` combined if `split_stderr` is disabled, only `stdout` otherwise
+- `exit_code`: exit code of command (integer)
+- `stderr`: stderr output if `split_stderr` is enabled, `None` otherwise
+
+A typical transition will then change
+
+```python
+(out, _) = run_cmd(cmd, log_all=True, simple=False)
+```
+
+to
+
+```python
+res = run_shell_cmd(cmd)
+out = res.output
+```
 
 ## Use cases
 
-...
+Examples:
+
+- Basic usage:
+  ```python
+  cmd = ' '.join([self.cfg['preinstallopts'], install_cmd, self.cfg['installopts']])
+  run_shell_cmd(cmd)
+  ```
+
+- Get error code for both failure and non-failure of the command, as otherwise `run_shell_cmd` will raise `RunShellCmdError`. Additionally, don't display this command in terminal output:
+  ```python
```

Comment:
```suggestion
- Get error code for both failure and non-failure of the command, as otherwise `run_shell_cmd` will raise `RunShellCmdError`. Additionally, don't display this command in terminal output:

  ```python
```